File: PHP/GuilhermeSuasPipasTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("GuilhermeSuasPipas.php");

class GuilhermeSuasPipasTest extends TestCase {

    public function testGuilhermeSuasPipas() {
        $obj = new GuilhermeSuasPipas();
        //Test 1
        $want = 2;
        $got = $obj->numberOfTwine(4);
        $this->assertEquals($want, $got);
        //Test 2
        $want = 35;
        $got = $obj->numberOfTwine(10);
        $this->assertEquals($want, $got);
        //Test 3
        $want = 44;
        $got = $obj->numberOfTwine(11);
        $this->assertEquals($want, $got);
    }

}

File: JAVA/GuilhermeSuasPipasTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class GuilhermeSuasPipasTest {

	@Test
	public void testGuilhermeSuasPipas() throws Exception {
		//Test 1
		int want = 2;
		int got = GuilhermeSuasPipas.numberOfTwine(4);
		assertEquals(want, got);
		//Test 2
		want = 35;
		got = GuilhermeSuasPipas.numberOfTwine(10);
		assertEquals(want, got);
		//Test 3
		want = 44;
		got = GuilhermeSuasPipas.numberOfTwine(11);
		assertEquals(want, got);
	}

}
